Doc/library/sqlite3.rst

```diff
@@ -406,14 +406,23 @@ Connection Objects
 
    .. attribute:: isolation_level
 
-      Get or set the current default isolation level. :const:`None` for autocommit mode or
-      one of "DEFERRED", "IMMEDIATE" or "EXCLUSIVE". See section
-      :ref:`sqlite3-controlling-transactions` for a more detailed explanation.
+      This attribute controls the :ref:`transaction handling
+      <sqlite3-controlling-transactions>` performed by ``sqlite3``.
+      If set to :const:`None`, transactions are never implicitly opened.
+      If set to one of ``"DEFERRED"``, ``"IMMEDIATE"``, or ``"EXCLUSIVE"``,
+      the strings representing the underlying SQLite `transaction behaviour`_,
+      transactions are implicitly opened before some DML statements.
```

Comment:
It seems this is the only use of the term "DML statement" in this document, and neither is it defined in the linked SQLite docs. I'm no database expert, though I have read most of these docs, but still had to go Google it myself and read through the linked resources to determine what statements were affected (with some ambiguity about `SELECT`).

Unless nearly all readers would be expected to know what it means and what statements (presumably `SELECT`, `INSERT`, `UPDATE` and `DELETE`) are included, I suggest explicitly specifying _which_ statements, and/or linking to a definition/explanation of the term.

Reply:
Hm, how about just simplifying it and linking to the explanation?

```suggestion
      some implicit :ref:`transaction management
      <sqlite3-controlling-transactions>` is performed.
```

Reply:
Looks good overall, I just suggest eliding "some" since it is rather vaugue and confusing, and sounds strange in that context in English technical writing:

```suggestion
      implicit :ref:`transaction management
      <sqlite3-controlling-transactions>` is performed.
```